backend/tarsy/integrations/mcp/client.py

```
@@ -47,13 +47,22 @@ async def initialize(self):
                 continue
                 
             try:
+                logger.debug(f"Initializing MCP server '{server_id}' with configuration:")
```

Ruff (0.12.2)

Line too long (90 > 88)

(E501)